package/libpiksi/libpiksi/src/util.c

```diff
@@ -99,6 +113,123 @@ bool device_is_duro(void)
                  strlen(DEVICE_DURO_ID_STRING)) == 0);
 }
 
+static bool device_has_ins(void)
+{
+  return access(SMOOTHPOSE_LICENSE_FILE_PATH, F_OK) != -1;
```

Comment:
Is there any chance of a permissions issue here?

Reply:
nope, this api simply checks for file existence, I have also since added a check for the existence of the posedaemon binary

Reply:
added a check for the posedaemon as well, since ideally they would both be there